firebase-storage/src/test/java/com/google/firebase/storage/network/MockConnectionFactory.java

```diff
@@ -166,17 +163,15 @@ public synchronized HttpURLConnection createInstance(@NonNull URL url) throws IO
     currentRecord++;
     if (currentRecord == pauseRecord) {
       Mockito.doAnswer(
-              new Answer<Void>() {
-                @Override
-                public Void answer(InvocationOnMock invocation) {
-                  try {
-                    pauseSemaphore.acquire();
-                  } catch (InterruptedException e) {
-                    e.printStackTrace();
-                  }
-                  return null;
-                }
-              })
+              (Answer<Void>)
```

Comment:
Would it compile without this line?

If it does, let's remove it.

Reply:
It does! I learned something today.